template/html/about.html

```diff
@@ -23,4 +23,15 @@ <h3>{{ t "page.about.credits" }}</h3>
     </ul>
 </div>
 
+{{ if .showGlobalConfigOptions }}
```

Comment:
You can simplify by just checking if the user is admin `user.IsAdmin`. The template variable is not needed anymore.